src/Umbraco.SampleSite.Web/Views/Partials/blockgrid/items.cshtml

```diff
@@ -0,0 +1,39 @@
+﻿@using Umbraco.Cms.Core.Models.Blocks
+@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<IEnumerable<BlockGridItem>>
+@{
+    if (Model?.Any() != true) { return; }
+}
+
+<div class="umb-block-grid__layout-container">
+    @foreach (var item in Model)
+    {
+        bool attrForceLeft = item.ForceLeft;
+        bool attrForceRight = item.ForceRight;
```

Comment:
Think we have to update this for when RC5 comes out since `ForceLeft` and `ForceRight` has been removed: https://github.com/umbraco/Umbraco-CMS/pull/13400

Reply:
Nice spot!